examples/react-dapp/src/ui/organisms/section.tsx

```diff
@@ -5,13 +5,14 @@ import { Text } from "../atoms/text"
 import { useMemo, useState } from "react"
 import { toast } from "react-toastify"
 import { isValidJSON } from "../../utils/json"
-import "react-toastify/dist/ReactToastify.css"
 import { Button } from "../atoms/button"
 import { CodeSection } from "../molecules/code-section"
 import { useIdentityKit } from "@nfid/identity-kit/react"
 import { getRequestObject } from "../../utils/requests"
 import { DropdownSelect } from "../molecules/dropdown-select"
 
+import "react-toastify/dist/ReactToastify.css"
```

Comment:
please ask egor and artem if we are okay going with default styles.
We modified them before in other products.